src/Backend/Modules/Slideshows/Actions/Add.php

```diff
@@ -80,4 +85,27 @@ private function handleForm()
             }
         }
     }
+
+    /**
+     * @return array
+     */
+    private function getPossibleTemplates()
+    {
+        $templates = array();
+        $finder = new Finder();
+        $finder->name('*.html.twig');
+        $finder->in(FRONTEND_MODULES_PATH . '/Slideshows/Layout/Widgets');
+        // if there is a custom theme we should include the templates there also
+        if (Theme::getTheme() != 'core') {
```

Comment:
This check is pretty useless I think. If someone wants to make a 'core' theme, the extra templates won't be loaded. 

In Fork 5 the default theme will be `Fork` so people should be able to expand on that